packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -9,8 +9,8 @@ import { ButtonSelect } from '../Dropdown/ButtonSelect';
 import { ToolbarButtonVariant, ToolbarButton } from '../ToolbarButton';
 
 // Default intervals used in the refresh picker component
+// TODO: How to translate these?
```

Comment:
**Question:** couldn't we just translate these the same way as the other props? 

I mean there is already an `intervals` prop (array of strings), couldn't the call-site just push the translated versions of strings in using this prop (optionally)?